dist/backdraft.js

```diff
@@ -532,6 +532,20 @@ _.extend(Plugin.factory, {
       return !columnConfig.bulk;
     });
 
+    // read initial filters from URL
+    var urlParamString = window.location.href.split("?")[1];
+    if (urlParamString && $.deparam(urlParamString) && ($.deparam(urlParamString).filter_json || $.deparam(urlParamString).ext_filter_json) ) {
```

Comment:
There are three places in this file where the fetching of the filter params is repeated. Can you dry that up? (It will probably read easier that way too.)